public/userview.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html lang="en">
+    <form action="http://localhost:8765" method="post">
+        <label for="img">Select image:</label>
+        <input type="file" id="img" name="img" accept="image/*">
```

Comment:
if you could make a preview feature that would be perfect, [here](https://stackoverflow.com/a/27165977/7267809) is how to do it (use the `FileReader` API)